### DISCUSSION TITLE
Error 500

### DISCUSSION THREAD

**Discussion by narvaldreams** [JulianPrieber/littlelink-custom#821] [General]
Good morning
I am trying to log into my account and for some reason my credentials are not validated. ***When I try to recover my password***, the server slows down and sends me a **500 error**.
Do you have time to resolve this issue? I have to create several credentials and this is delaying me.
![Captura de pantalla de 2024-09-26 17-49-31](https://github.com/user-attachments/assets/cbaebf75-4c18-4303-a74a-e140eab435aa)

**Comment by JulianPrieber** [comment]
Please switch your mail server type from built-in to smtp
